lib/smart_app_launch/client_suite/authorization_request_verification_test.rb

```diff
@@ -12,6 +12,18 @@ class SMARTClientAppLaunchAuthorizationRequestVerification < Inferno::Test
     description %(
       Check that SMART authorization requests made are conformant.
     )
+    verifies_requirements 'hl7.fhir.uv.smart-app-launch_2.2.0@3',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@32',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@33',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@34',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@35',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@37',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@39',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@40',
```

Comment:
@39 and @40 should be attestations. Remove from here.

@39
[When] the app constructs a request for an authorization code [the] …  `state` [parameter is] `required` [and] The parameter SHALL be used for preventing cross-site request forgery or session fixation attacks

@40
[When] the app constructs a request for an authorization code [the] …  `state` [parameter is] `required` [and] ... The app SHALL use an unpredictable value for the state parameter with at least 122 bits of entropy (e.g., a properly configured random uuid is suitable).

Reply:
Removed in [9b4d4ad](https://github.com/inferno-framework/smart-app-launch-test-kit/pull/105/commits/9b4d4adf0a9945b994d01c6ee21808ba05be91a6).